### ISSUE TITLE
InstanceUrl field breaks Salesforce validation

### ISSUE THREAD

**Issue by jknetl** [syndesisio/syndesis-verifier#17]
When you try to verify a salesforce connection which has "instance url" field defined, the validation will always fail with `Response with status: 500 Internal Server Error for URL: https://syndesis.192.168.42.95.xip.io/api/v1/connectors/salesforce/verifier`.

![instanceurl-parameter](https://user-images.githubusercontent.com/5871202/29617604-b4c645d6-8815-11e7-8787-dd68a129aa85.png)

The problem is that instanceUrl option is not known by the salesforce verifier. This statement can be found in verifier log:

```
2017-08-23 13:07:59.587 ERROR 1 --- [  XNIO-3 task-6] i.s.verifier.impl.SalesforceVerifier     : salesforce -->
2017-08-23 13:07:59.587 ERROR 1 --- [  XNIO-3 task-6] i.s.verifier.impl.SalesforceVerifier     :    UNKNOWN_PARAMETER : Unknown option instanceUrl
```

If you just delete the instance url option from the form the validation will succeed afterwards.

**Comment by rhuss** [comment]
@jknetl thanks for the report. Yes, we remarked this, too and afaik there is a projected fix for this (cc @zregvart )

**Comment by zregvart** [comment]
This should be fixed with syndesisio/syndesis-rest#523 I'm inclined to leave it as is until then, a quick workaround is to empty the _Instance URL_ field.

**Comment by zregvart** [comment]
This was fixed.
